packages/directory/package.json

```diff
@@ -18,12 +18,14 @@
   "scripts": {
     "build": "ember build --environment=production",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:js": "eslint . && tsc --noEmit",
+    "lint": "npm run lint:hbs && npm run lint:js",
     "start": "ember serve",
-    "test": "export COVERAGE=true; ember test --test-port 0"
+    "test": "export COVERAGE=true; ember test --test-port 0",
+    "prepublishOnly": "ember ts:precompile",
```

Comment:
Have you executed `ember ts:precompile` yet? I had to change `test-helpers.js` to `test-heplers.ts` for `qunit-dom` to work.